WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuElement.swift

```diff
@@ -1,17 +1,31 @@
 import SwiftUI
 
+enum HubMenuBadgeType {
+    case newFeature
```

Comment:
How about renaming this to something like `PaymentsBadge` or similar?

Reply:
As we talked before, I wanted to keep it as generic as possible, so in case we add a new button we can use the badge without having to rename it (I presume we would use the same red dot badge). Because of that, I preferred to use `newFeature` to make it more abstract.

Reply:
Yeah! That makes sense, I reviewed this one before reading your comments in the other PR :D